main/wine/template.py

```diff
@@ -68,32 +67,49 @@
 options = ["!lto", "!check"]
 
 
+match self.profile().arch:
+    case "x86_64":
+        configure_args += ["--enable-archs=x86_64,i386"]
+
+
 def post_install(self):
     self.install_link("usr/bin/wine64", "wine")
 
 
 @subpackage("wine-devel")
 def _(self):
-    # llvm-strip/objcopy cannot handle windows .a's
-    self.nostrip_files = [
-        "usr/lib/wine/i386-windows/*.a",
-        "usr/lib/wine/x86_64-windows/*.a",
+    extra_files = [
+        "usr/bin/function_grep.pl",
+        "usr/bin/widl",
+        "usr/bin/winebuild",
+        "usr/bin/winecpp",
+        "usr/bin/winedbg",
+        "usr/bin/winedump",
+        "usr/bin/wineg++",
+        "usr/bin/winegcc",
+        "usr/bin/winemaker",
+        "usr/bin/wmc",
+        "usr/bin/wrc",
     ]
-    return self.default_devel(
-        extra=[
-            "usr/bin/function_grep.pl",
-            "usr/bin/widl",
-            "usr/bin/winebuild",
-            "usr/bin/winecpp",
-            "usr/bin/winedbg",
-            "usr/bin/winedump",
-            "usr/bin/wineg++",
-            "usr/bin/winegcc",
-            "usr/bin/winemaker",
-            "usr/bin/wmc",
-            "usr/bin/wrc",
-            "usr/lib/wine/i386-windows/*.a",
-            "usr/lib/wine/x86_64-unix/*.a",
-            "usr/lib/wine/x86_64-windows/*.a",
-        ]
-    )
+
+    # llvm-strip/objcopy cannot handle windows .a's
+    match self.rparent.profile().arch:
```

Comment:
why do all this stuff? you can just change the wildcard to a looser one, like `"usr/lib/wine/*/*.a"`

Reply:
ok, understood. I tried to follow the syntax in other templates.